stellargraph/data/explorer.py

```diff
@@ -900,8 +900,10 @@ def _sample(self, n, biases, np_rs):
 
     def _exp_biases(self, times, t_0, decay):
         # t_0 assumed to be smaller than all time values
-        dist = np.exp(t_0 - np.array(times) if decay else np.array(times) - t_0)
-        sum_dist = np.sum(dist)
+        relative_times = t_0 - np.array(times) if decay else np.array(times) - t_0
+        dist = np.exp(relative_times - relative_times.max())
+        sum_dist = dist.sum()
+
         return dist / sum_dist
```

Comment:
Ah, given this whole function is computing a softmax, I wonder if it would indeed be better to just do `scipy.special.softmax(relative_times)` as you'd originally had it? 

My suggestion on #1010 was because I thought we were just computing an arbitrary non-normalised array where the ratios matter, but not the specific values, however, it turns out we are computing a normalised vector.

Reply:
Ah I see, I got a bit confused too. Yeah okay I'll switch back since it's less to read